changelog.d/9358.misc

```diff
@@ -0,0 +1 @@
+Added a fix that invalidates cache for empty timed-out sync responses.
```

Comment:
shouldn't this have been a `.bugfix`?

Reply:
shoot, yes, could you change the file extension in the `develop` branch?

Reply:
Ah, it's already suggested in #9522 to include it there, i'll resolve this then :+1: